### PR TITLE
LBSD-627 Administrator should be able to edit blog settings

### PR DESCRIPTION
**Pull request by vladnicoara**
Administrator should be able to edit blog settings even if not blog owner

### CODE REVIEW

File: client/app/scripts/liveblog-edit/module.js [hunk 1]
```diff
@@ -462,10 +462,10 @@ define([
             return privileges.userHasPrivileges({'publish_post': 1});
         }
         function isUserOwner(archive) {
-            if ($rootScope.currentUser._id !== archive.original_creator) {
-                return false;
-            } else {
+            if ($rootScope.currentUser._id === archive.original_creator || $rootScope.currentUser.user_type === 'administrator') {
```
Comment (vied12) [inline client/app/scripts/liveblog-edit/module.js]: @vladnicoara You created a `SecurityService` somewhere, isn't it ? :)
What about using it to check the permission ?

Comment (vladnicoara) [inline client/app/scripts/liveblog-edit/module.js]: This is the SecurityService, I am modifying one of the methods there

Comment (vied12) [inline client/app/scripts/liveblog-edit/module.js]: Oh yes it is... :yum: sorry
:+1:

**Comment by vied12** [comment]
LGTM

**Pull request merged**